ubuntu/entrypoint.sh

```diff
@@ -80,6 +84,84 @@ function unload_modules() {
     done
 }
 
+
+# find all available vfs
+# return format <pf_name> <vf_name> <vf_index> <vf_mac> <vf_pci_addr>
+function find_mlx_vfs() {
+    for ethpath in /sys/class/net/*; do
+        if (grep $VENDOR "$ethpath"/device/vendor >/dev/null 2>&1); then
+            for vf in "$ethpath"/device/virtfn[0-9]*/net/*; do
+                [[ -d $vf ]] && [[ -e $vf/address ]] && [[ -L $vf/device ]] || continue
```

Comment:
is this check necessary ? you can rely on the layout of sysfs.

Reply:
Yes, I think we can safely drop `[[ -e $vf/address ]] && [[ -L $vf/device ]]` check. Done